### ISSUE TITLE
android list suggestion

### ISSUE THREAD

**Issue by demogit-code** [eylenburg/eylenburg.github.io#42]
maybe add shiftphone to the list
on https://eylenburg.github.io/android_comparison.htm ?

--

shiftphone seems to have a similar, (not the same) , idealogy as fairphone
https://www.shift.eco/en/

--

eg https://calyxos.org/install/devices/axolotl/linux/

**Comment by eylenburg** [comment]
Yes, I completely agree. The Shiftphone 6mq seems to be supported by CalyxOS, LineageOS, /e/ and IodeOS, but not DivestOS and GrapheneOS. However, this device is very old now and will be superseded by the Shiftphone 8 soon. I think I'll wait until that's released before adding it to the table.
